xml/ha_rear.xml

```diff
@@ -553,6 +553,20 @@ POST_RECOVERY_SCRIPT=( 'if snapper --no-dbus -r $TARGET_FS_ROOT get-config | gre
     <command>man rear</command>, section <citetitle>BACKUP SOFTWARE INTEGRATION</citetitle>.
    </para>
   </example>
+
+  <example xml:id="ex-ha-rear-config-multipath">
+    <title>Backing up multipath devices</title>
+    <para>
+     By default, &rear; ignores file systems on <filename>/dev/mapper</filename> devices because
+     it cannot re-create them when restoring the system. You can allow &rear; to include
+     these file systems by adding the following line to <filename>/etc/rear/local.conf</filename>:
+    </para>
+    <screen>AUTOEXCLUDE_MULTIPATH=n</screen>
+    <para>
+     Before you can restore the backup, you must manually re-create the device structure on
```

Comment:
This is very vague - what does it mean exactly?

And, I did have `AUTOEXCLUDE_MULTIPATH=n` during back and during restore I did nothing regarding to MPIO device and `rear -d -D rescue` has started and I see:

``` shell
RESCUE localhost:~ # findmnt -t nfs4
TARGET                                 SOURCE                                            FSTYPE OPTIONS                o
/var/tmp/rear.jIMhs1C5TnL12uu/outputfs 192.168.252.1:/var/lib/libvirt/images/jbelka/rear nfs4   ro,relatime,vers=4.0,rsize=1048576,wsize=1048576,namlen=255,hard,proto=tcp,timeo=600,retrans=2,sec=sys,clientaddr=192.168.252.140,local_lock=no
RESCUE localhost:~ # lsblk
NAME                        MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINTS
sda                           8:0    0    25G  0 disk  
|-sda1                        8:1    0   512M  0 part  
|-sda2                        8:2    0  24.5G  0 part  
`-35000039afc38015c         254:0    0    25G  0 mpath 
  |-35000039afc38015c-part1 254:1    0   512M  0 part  /mnt/local/boot/efi
  `-35000039afc38015c-part2 254:2    0  24.5G  0 part  
    `-system-root           254:3    0    24G  0 lvm   /mnt/local/var
                                                       /mnt/local/tmp
                                                       /mnt/local/usr/local
                                                       /mnt/local/root
                                                       /mnt/local/boot/grub2/x86_64-efi
                                                       /mnt/local/srv
                                                       /mnt/local/opt
                                                       /mnt/local/home
                                                       /mnt/local/boot/grub2/i386-pc
                                                       /mnt/local/.snapshots
                                                       /mnt/local
sdb                           8:16   0    25G  0 disk  
|-sdb1                        8:17   0   512M  0 part  
|-sdb2                        8:18   0  24.5G  0 part  
`-35000039afc38015c         254:0    0    25G  0 mpath 
  |-35000039afc38015c-part1 254:1    0   512M  0 part  /mnt/local/boot/efi
  `-35000039afc38015c-part2 254:2    0  24.5G  0 part  
    `-system-root           254:3    0    24G  0 lvm   /mnt/local/var
                                                       /mnt/local/tmp
                                                       /mnt/local/usr/local
                                                       /mnt/local/root
                                                       /mnt/local/boot/grub2/x86_64-efi
                                                       /mnt/local/srv
                                                       /mnt/local/opt
                                                       /mnt/local/home
                                                       /mnt/local/boot/grub2/i386-pc
                                                       /mnt/local/.snapshots
                                                       /mnt/local
sr0                          11:0    1 521.2M  0 rom   
```

Thus, the claim that "you must manually re-create..." is invalid. Please, clarify.
(Yes, my system was the same for the restore, thus WWN for MPIO was the one used during the backup.)

Reply:
I just tried with a MPIO device where I just changed WWN, and I got:

```
...
Setting up multipathing
Activating multipath
multipath activated
Starting multipath daemon
multipathd started
Listing multipath device found
35000039afc38015d dm-0 QEMU,QEMU HARDDISK size=25G
Comparing disks
Device mapper!35000039afc38015c does not exist (manual configuration needed)
Switching to manual disk layout configuration (GiB sizes rounded down to integer)
/dev/mapper/35000039afc38015c had size 26843545600 (25 GiB) but it does no longer exist
/dev/dm-0 was not used on the original system and has now 26843545600 (25 GiB)
/dev/dm-1 was not used on the original system and has now 536870912 (0 GiB)
/dev/dm-2 was not used on the original system and has now 26305609216 (24 GiB)
Using /dev/mapper/35000039afc38015d (same size 26843545600) for recreating /dev/mapper/35000039afc38015c
Current disk mapping table (source => target):
  /dev/mapper/35000039afc38015c => /dev/mapper/35000039afc38015d

UserInput -I LAYOUT_MIGRATION_CONFIRM_MAPPINGS needed in /usr/share/rear/layout/prepare/default/300_map_disks.sh line 322
Confirm or edit the disk mapping
1) Confirm disk mapping and continue 'rear recover'
2) n/a
3) Edit disk mapping (/var/lib/rear/layout/disk_mappings)
4) Use Relax-and-Recover shell and return back to here
5) Abort 'rear recover'
(default '1' timeout 300 seconds)
```
It seems to work based on `disklayout.conf` modification.

``` shell
rear> diff -uNp /var/lib/rear/layout/disklayout.conf{.20240802164243.recover.630.orig,} | grep -Pv '^(-|\+|\s*)?#'
--- /var/lib/rear/layout/disklayout.conf.20240802164243.recover.630.orig        2024-08-01 15:17:51.000000000 +0200
+++ /var/lib/rear/layout/disklayout.conf        2024-08-02 16:52:29.683634297 +0200
@@ -3,21 +3,21 @@
-lvmdev /dev/system /dev/mapper/35000039afc38015c-part2 b7y0ip-8pbs-4O1i-f727-UqaD-7yCT-Tles0d 51378143
+lvmdev /dev/system /dev/mapper/35000039afc38015d-part2 b7y0ip-8pbs-4O1i-f727-UqaD-7yCT-Tles0d 51378143
 lvmgrp /dev/system 4096 6271 25686016
@@ -26,7 +26,7 @@ lvmgrp /dev/system 4096 6271 25686016
 lvmvol /dev/system root 25769803776b linear 
-fs /dev/mapper/35000039afc38015c-part1 /boot/efi vfat uuid=1FC2-0570 label= options=rw,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro
+fs /dev/mapper/35000039afc38015d-part1 /boot/efi vfat uuid=1FC2-0570 label= options=rw,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro
 fs /dev/mapper/system-root / btrfs uuid=f733719f-9f7b-4b28-93bf-97e53351fbed label='ROOT' options=rw,relatime,space_cache,subvolid=259,subvol=/@/.snapshots/1/snapshot
@@ -93,8 +93,8 @@ btrfsmountedsubvol /dev/mapper/system-ro
 btrfsnocopyonwrite @/var
-multipath /dev/mapper/35000039afc38015c 26843545600 gpt /dev/sda,/dev/sdb
-part /dev/mapper/35000039afc38015c 536870912 1048576 rear-noname boot,esp /dev/mapper/35000039afc38015c-part1
-part /dev/mapper/35000039afc38015c 26305609216 537919488 rear-noname lvm /dev/mapper/35000039afc38015c-part2
+multipath /dev/mapper/35000039afc38015d 26843545600 gpt /dev/sda,/dev/sdb
+part /dev/mapper/35000039afc38015d 536870912 1048576 rear-noname boot,esp /dev/mapper/35000039afc38015d-part1
+part /dev/mapper/35000039afc38015d 26305609216 537919488 rear-noname lvm /dev/mapper/35000039afc38015d-part2
```

Reply:
That was based on the comments in https://bugzilla.suse.com/show_bug.cgi?id=1202694, which don't go into much detail about what to do (fair, since it's outside the scope of the bug). Thanks for looking into it and providing some info.

Reply:
Looking at this again, I think since ReaR itself explains what to do when this comes up, I don't need to get too detailed about it in the doc.